examples/most-redis-docker/README.md

```diff
@@ -0,0 +1,124 @@
+# Most + Redis + Docker
+
+A suite of simple apps to illustrate how to use most to handle redis events.
+
+This suite may be run using [docker](https://www.docker.com) or [node.js
+10.x](https://nodejs.org/en/).  If you have docker installed already or would
+like to try it, follow the docker instructions.  Otherwise, you will need to be
+running redis locally.  Follow the node instructions to install node and redis.
+
+## Install
+
+### Pre-requisites
+
+1. Install [git](https://help.github.com/articles/set-up-git/)
+2. Clone this repo:
+
+```sh
+git clone git@github.com:mostjs/core.git
+cd examples/most-redis-docker
+```
+
+### Assumptions
+
+The remainder of this document assumes your working directory is
+`most-redis-docker`.  You may have to modify some of the following directions
+if you chose another folder name.
+
+This project uses environment variables.  If you are using the windows command
+prompt, you may have to change some commands slightly to use env vars. For
+example, rather than type `REDIS_MYCHANNEL=my_channel npm run consume`, you
+would type `set REDIS_MYCHANNEL=my_channel&&npm run consume`.
+
+### Option 1: Docker
+
+1.  Install [docker](https://www.docker.com/get-started).  (You will need to
+    create a [DockerHub](https://hub.docker.com) account.)
+2.  Build the docker image for this project:
+
+```sh
+docker build -t most-redis .
+```
+
+### Option 2: node.js
+
+1. Install [node 10.x](https://nodejs.org/en/) or the latest LTS version.
+2. Install the dependencies for this project:
+
+```sh
+npm install
+```
+
+3. Install [redis](https://redis.io/topics/quickstart).
+
+If you already have an instance of redis running, you may edit the docker-
+compose.yml file in the `most-redis-docker` folder.  Change any values for
+`REDIS_HOST` and `REDIS_PORT` according to your redis instance
+
+## Run
+
+When running this example, the publish service periodically sends JSON messages
+to redis, and the consume services receive the JSON messages and transform
+them.  In your terminal(s), you should expect to see informational logs from
+each service as they start up and as they process the JSON messages.
+
+### Option 1: Docker
+
+1.  From the `most-redis-docker` folder, create and run the docker-compose
+    network defined in `docker-compose.yml`:
+
+```sh
+docker-compose up
```

Comment:
Similarly, would making `npm start` run this be helpful, or no?

Reply:
See other comment ... docker/docker-compose is ok.

> I wonder if we could start three instances of node from one `npm script`?

I'm sure there's a way ... although windows will probably cause problems.  Maybe something like [npm-run-all](https://www.npmjs.com/package/npm-run-all) can help.